bellows/types/named.py

```diff
@@ -1405,6 +1406,14 @@ class EmberKeyStatus(basic.uint8_t, enum.Enum):
     TC_FAILED_TO_SEND_APP_KEYS = 0x0F
     TC_FAILED_TO_STORE_APP_KEY_REQUEST = 0x10
     TC_REJECTED_APP_KEY_REQUEST = 0x11
+    TC_FAILED_TO_GENERATE_NEW_KEY = 0x12
```

Comment:
Are these from a new version of EZSP? Any other changes from that version that should go into bellows?

Reply:
Have you actually seen these responses from the NCP?